### DISCUSSION TITLE
Prevent close dialog when click ok

### DISCUSSION THREAD

**Discussion by matamune94** [quasarframework/quasar#17233] [Quasar App Extensions] [answered]
I'm using the Dialog plugin, is there any way to prevent the dialog from closing when I press the ok button?

**Comment by OZ1245** [comment]
Same question. I have a dialog and a request. I want to close the dialog after successful status. I tried returning false from the request function, but it doesn't work - the dialog closes.

**Comment by dongwa** [comment]
I have submitted a PR for this issue, but it hasn't received any attention for nearly six months. Here is the link to the pull request: [PR #17121 - Fix for Multi-Select Issue in Table Component](https://github.com/quasarframework/quasar/pull/17121).

**Comment by rstoenescu** [comment, marked as answer]
Hello,

We have explained numerous times that the default Dialog plugin should cover most of the user-cases, but definitely not all possible ones. Should one need some custom behavior that is not currently supplied, [a custom component](https://quasar.dev/quasar-plugins/dialog#invoking-custom-component) should be created and invoked instead.
The alternative is to open the Pandora's box and allow all of the requests and end up with a bloated + complex + non-efficient + hard to grasp component
